package.json

```diff
@@ -40,7 +40,7 @@
     "decompress": "^4.2.0",
     "deep-equal": "^1.0.1",
     "ejs": "^2.5.8",
-    "embarkjs": "^0.3.4",
+    "embarkjs": "latest",
```

Comment:
Doesn't this risk installations of a tagged/published versions of embark behaving differently over time as `embarkjs` changes over time?

Reply:
I'll change once embarkjs is merged